phpstan.neon.dist

```diff
@@ -12,6 +12,9 @@ parameters:
         # Ignore proxy manager magic
         - '~ProxyManager\\Proxy\\VirtualProxyInterface~'
         - '~Variable method call on Doctrine\\Migrations\\AbstractMigration~'
+        -
+            message: '~^Call to function in_array\(\) requires parameter #3 to be true\.$~'
```

Comment:
Out of curiosity, is there a reason why we can't enable strict mode on the `in_array` call?

Reply:
the reason of this is explained in https://github.com/doctrine/migrations/pull/891/files#diff-11b21ce809b5364acce6a5fd2482f280R54